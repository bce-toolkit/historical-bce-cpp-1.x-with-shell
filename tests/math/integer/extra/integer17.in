389973922125897709480480351814841605868503716612609891644787324462433569418481169804696724174093875215081730024401896043017110
277867468323694297996538732941740840992363847510833660567129098065662940512556527318023811102310270011886
