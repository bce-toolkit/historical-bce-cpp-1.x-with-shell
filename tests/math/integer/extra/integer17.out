389973922125897709480758219283165300166500255345551632485779688309944403079048298902762387114606431742399753835504206313028996
389973922125897709480202484346517911570507177879668150803794960614922735757914040706631061233581318687763706213299585773005224
108361066453384708719839993582396781702068681480137590742375020104857269254050228643002389983683737004160402170227203942445010267972479602084637219299937741516162027457396621994430561409144101895306150073217276149263075087001369460
1403452964388072904604
2796227681459226826747780966020523349916797752641919389457081121759769935445103552638595965778418893966
0
1
0
0
1
1
